lib/ros/be/infra/files/terraform/aws/s3/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region  = "us-east-1"
+  profile = var.aws_profile
+  alias   = "us-east-1"
+}
+
+resource "aws_s3_bucket" "this" {
+  count = length(var.s3_bucket_names)
+  bucket = var.s3_bucket_names[count.index]
+  acl    = "private"
+
+  tags = var.tags
+}
+
```

Comment:
I think we don't need to have a separate aws provider section. Unlike cloudfront, there's no need to create a separate resource on a specific region.

Reply:
fixed